src/XKit/XKitCompat.cs

```diff
@@ -0,0 +1,44 @@
+#if !XAMCORE_4_0
+using System;
+using Foundation;
+using ObjCRuntime;
+
+#if MONOMAC
+using AppKit;
+#else
+using UIKit;
+#endif
+
+#if MONOMAC
+namespace AppKit {
+#else // macOS
+namespace UIKit {
+#endif // iOS, tvOS, WatchOS
+
+#if !WATCH
+
+#if !COREBUILD
+
+	[iOS (7,0)]
+	public partial class NSLayoutManager {
+		[Obsolete ("Always throws NotSupportedException (). (not a public API).")]
+		public virtual void ReplaceTextStorage (NSTextStorage newTextStorage)
+			=> throw new NotSupportedException ();
+
+
+		[Obsolete ("Always throws NotSupportedException (). (not a public API).")]
```

Comment:
It's public for macOS, but nowhere else:

https://developer.apple.com/documentation/appkit/nslayoutmanager/1403037-settemporaryattributes?language=objc

Reply:
This hasn't been fixed.